docs/source/yosys_internals/index.rst

```diff
@@ -0,0 +1,40 @@
+.. _chapter:overview:
+
+Yosys internals
+===============
+
+.. todo:: less academic
+
+Yosys is an extensible open source hardware synthesis tool. It is aimed at
+designers who are looking for an easily accessible, universal, and
+vendor-independent synthesis tool, as well as scientists who do research in
+electronic design automation (EDA) and are looking for an open synthesis
+framework that can be used to test algorithms on complex real-world designs.
+
+Yosys can synthesize a large subset of Verilog 2005 and has been tested with a
+wide range of real-world designs, including the `OpenRISC 1200 CPU`_, the
```

Comment:
A lot of more up-to-date examples we could put here (at least one RISC-V core mention seems customary nowadays)